gradle-mvn-push.gradle

```diff
@@ -105,6 +105,11 @@ afterEvaluate { project ->
 
   signing {
     required { isReleaseBuild() && gradle.taskGraph.hasTask("uploadArchives") }
+    if (project.hasProperty('signing.inMemoryKey')) {
+      def signingKey = findProperty("signing.inMemoryKey").replace("\\n","\n")
+      def signingPassword = findProperty("signing.password")
+      useInMemoryPgpKeys(signingKey, signingPassword)
```

Comment:
Why aren't we passing `keyId` as well, like we [do in amplify-android](https://github.com/aws-amplify/amplify-android/blob/main/configuration/publishing.gradle#L109)?

Reply:
The version of maven for that repo doesn't support that particular overload. My understanding is that it allows you to specify a subkey (if desired) to sign your artifacts with [1]

[1] https://github.com/gradle/gradle/pull/10366